geomesa-geojson/geomesa-geojson-rest/src/main/scala/org/locationtech/geomesa/geojson/servlet/GeoJsonServlet.scala

```diff
@@ -8,7 +8,11 @@
 
 package org.locationtech.geomesa.geojson.servlet
 
+import java.io.Closeable
+import java.util.concurrent.ConcurrentHashMap
+
 import org.json4s.{DefaultFormats, Formats}
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
```

Comment:
Is the ADS required for the GeoJSON support?  Or will this work with a general DataStore?